### PR TITLE
release v1.9.1 preparation

### PR DESCRIPTION
**Pull request by emasab**
Updated version and CHANGELOG

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,26 @@
 
 This is a feature release:
 
- * Built-in support for Mac OSX M1 / arm64.
+ * Schema Registry support for Avro (generic and specific), Protobuf and JSON Schema. (@rayokota, #776).
+   See:
+
+   [examples/avro_generic_consumer_example](examples/avro_generic_consumer_example)
```
[the remark this answers is not in the document]
Comment (mhowlett) [reply CHANGELOG.md:11]: oh, i see magnus suggested you link to one.
what i would do is do it in line, i.e.: [Avro](examples/avro_generic_consumer_example) (generic and specific), [Protobuf](examples/...) and [JSON]](...) ...

[the remark this answers is not in the document]
Comment (emasab) [reply CHANGELOG.md:11]: otherwise I can put a "see the new examples in [examples](examples)"

[the remark this answers is not in the document]
Comment (emasab) [reply CHANGELOG.md:11]: yeah, I put them inline

[the remark this answers is not in the document]
Comment (emasab) [reply CHANGELOG.md]: I've linked only the producer examples

**Pull request merged**